tools/clang/test/DXILValidation/vector-validation.hlsl

```diff
@@ -0,0 +1,16 @@
+// RUN: %dxc -T vs_6_9 %s -Od | FileCheck %s
```

Comment:
Like with the other one, I think the RUN line makes it look like it's run as a test when it isn't.